IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignmentSegment/Documentation.md

```diff
@@ -0,0 +1,12 @@
+**Road domain definition**
```

Comment:
Would it be possible to generalize the definition?

"An _IfcAlignmentSegment_ is a segment of an _IfcAlignment_ where either the vertical direction or horizontal direction obey a unique mathematical description as a function of chainage where chainage is the longitudinal distance along the horizontal projection of the whole alignment.

Note 1 to entry: Unless otherwise stated, for railway, the appertaining track alignment design parameters are defined for the track centreline.

Note 2 to entry: For roads, the alignment typically describes the path of the road centerline, but in certain situations, e.g. ramps, the alignment may describe the path of other edges of the road body

&gt;NOTE Rail domain definitions according to EN 13803:2017"

I don't have a direct correspondence to EN 13803:2017 but will try to find something.

Do the notes belong here or in the IfcAlignment documentation?

Reply:
I will use this proposal with one minor modification.

"is a segment of an IfcAlignment where either the vertical direction or horizontal direction obey a unique"
-->
"is a segment of an IfcAlignment where either the vertical direction or horizontal direction and in the case of trackdesign also cant obey a unique"

Please expect a commit soon.

Reply:
Agree with this structure and the final tuning from Andreas